apps/base/utils.py

```diff
@@ -18,20 +17,17 @@ def paginated_queryset(queryset, request):
     return (paginator, result_page)
 
 
-@deconstructible
```

Comment:
Removing `@deconstructible` will not cause any impact.
But if we merge this PR into the production, then it will change the paths for the upcoming submissions that will be stored in the database, as currently all the submissions are being stored in only one folder.
I think we should deploy this after the current `VQA challenge` is over. What are your views about it?